server/models/chatroom.js

```diff
@@ -1,30 +1,27 @@
-const mongoose = require("mongoose");
-const User = require("./user");
-const Message = require("./message");
+const mongoose = require('mongoose');
```

Comment:
@astro0725 @PaBoba we keep flipping between `''` and `""`. We should pick one of these conventions and stick with it

Reply:
@PaBoba lmk what u prefer